### ISSUE TITLE
display mistake

### ISSUE THREAD

**Issue by Von-Darcy** [pengx17/logseq-plugin-bullet-threading#26]
<img width="95" alt="image" src="https://github.com/user-attachments/assets/f1f9aff2-bc21-4765-a0b7-3b5d691ff70f">
using the latest Logseq version 0.10.9, dev theme

**Comment by zimya** [comment]
same on my logseq 0.10.9 (83)

**Comment by punkyard** [comment]
<img width="216" alt="Capture d’écran 2024-10-29 à 03 16 42" src="https://github.com/user-attachments/assets/e701cbb4-fbd9-4190-a44c-78735e972600">

**Comment by sunday9th** [comment]
@punkyard If u still using this theme, might as well try following 

![Image](https://github.com/user-attachments/assets/a14468e8-7c98-4c83-b80c-2e84164b2191)

```js
body:has(> #logseq-bullet-threading_lsp_main) {
    & .block-children-left-border + .block-children {
        border-left-width: 0px !important; 
    }
}

.ls-block .ls-block > div > div.items-center::before {
    left: calc(var(--ls-block-bullet-threading-width)* -0.5) !important;  
}

.ls-block .block-children > .ls-block::before {
    left: calc(var(--ls-block-bullet-threading-width)* -0.5) !important;
}
```

**Comment by punkyard** [comment]
[](url)> [@punkyard](https://github.com/punkyard) If u still using this theme, might as well try following
> 
> ![Image](https://github.com/user-attachments/assets/a14468e8-7c98-4c83-b80c-2e84164b2191)
> 
> body:has(> #logseq-bullet-threading_lsp_main) {
>     & .block-children-left-border + .block-children {
>         border-left-width: 0px !important; 
>     }
> }
> 
> .ls-block .ls-block > div > div.items-center::before {
>     left: calc(var(--ls-block-bullet-threading-width)* -0.5) !important;  
> }
> 
> .ls-block .block-children > .ls-block::before {
>     left: calc(var(--ls-block-bullet-threading-width)* -0.5) !important;
> }

thanks i did - it's a bit better now ;)

<img width="212" alt="Image" src="https://github.com/user-attachments/assets/65392140-0386-42b4-afc0-40707a1b2895" />

**Comment by yixion** [comment]
I have the same problem. The problem happened when I didn't connect to the internet.
like this,

![Image](https://github.com/user-attachments/assets/807a4711-74ec-4348-9fa2-9f99e90b1fe9)

**Comment by giocomai** [comment]
Indeed, I have the same problem, *only* when I'm not online when Logseq starts... if I connect to the internet and restart Logseq, then it looks right again. 

hints about how to fix this?
